pkg/migration/controllers/migration.go

```diff
@@ -364,7 +366,37 @@ func (m *MigrationController) handle(ctx context.Context, migration *stork_api.M
 						err.Error())
 					err = m.updateMigrationCR(context.Background(), migration)
 					if err != nil {
-						log.MigrationLog(migration).Errorf("Error updating")
+						log.MigrationLog(migration).Errorf("Error updating CR, err: %v", err)
+					}
+					return nil
+				}
+			}
+			// Make sure if transformation CR is in ready state
```

Comment:
What if we did the dryRun and after a that a few resources got created? The migration could still potentially if a transform rule does not apply to this new resource.